index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <title>Marcelo Alarcon Portafolio</title>
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="images/codeoutlinedprogrammingsigns_81143.ico"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <img src="images/Welcome.png" alt="logo" />
+      <img src="images/Union.png" alt="menu button" />
```

Comment:
- [ ] I think it would be better to use a ```nav``` tag. :+1: 

    _The ```<nav>``` tag is for semantics reasons, and the right position for convention and user experience for mobile devices. :joy:_

Example: 
```
<nav>
   <img src="images/Union.png" alt="menu button" />
</nav>
```